website/content/docs/agent/telemetry.mdx

```diff
@@ -85,6 +85,7 @@ These are some metrics emitted that can help you understand the health of your c
 | `consul.raft.leader.lastContact` | Measures the time since the leader was last able to contact the follower nodes when checking its leader lease. | ms        | timer   |
 | `consul.raft.state.candidate`    | Increments whenever a Consul server starts an election.                                                   | elections | counter |
 | `consul.raft.state.leader`       | Increments whenever a Consul server becomes a leader.                                                     | leaders   | counter |
+| `consul.server.is_leader`        | Track if a server is a leader.                                                                            | 1 or 0    | gauge   |
```

Comment:
thanks for adding docs!

This metrics seems pretty important and convenient. 

I wonder if there is value in having a section dedicated to it even before:

* https://www.consul.io/docs/agent/telemetry#transaction-timing

You could also do the docs in a follow up PR